compiler/next/lib/parsing/chpl.ypp

```diff
@@ -2531,33 +2524,41 @@ var_decl_stmt_inner:
 tuple_var_decl_component:
   TUNDERSCORE
     {
-      auto varDecl = Variable::build(BUILDER, LOC(@$),
-                                     $1, context->visibility,
+      auto varDecl = Variable::build(BUILDER, LOC(@$), $1,
+                                     context->visibility,
+                                     context->linkage,
+                                     context->consumeVarDeclLinkageName(),
                                      context->varDeclKind,
                                      context->isVarDeclConfig,
                                      context->currentScopeIsAggregate(),
-                                     nullptr, nullptr);
+                                     /*typeExpression*/ nullptr,
+                                     /*typeExpression*/ nullptr);
```

Comment:
Shouldn't this second one be "initExpression" like below?

Reply:
Lol I must have been going at warp speed when I copied that line. Thanks!!!